tasks/ta_processor.py

```diff
@@ -128,32 +129,43 @@ def process_individual_upload(
             return False
         else:
             flaky_test_set = get_flake_set(db_session, upload.report.commit.repoid)
-            pg = PGDriver(db_session, flaky_test_set)
-            bq_enabled = False
-            if get_config("services", "bigquery", "enabled", default=False):
-                bq = BQDriver()
-                bq_enabled = True
-
-            for parsing_info in parsing_infos:
-                framework = parsing_info["framework"]
-                testruns = parsing_info["testruns"]
-                pg.write_testruns(
-                    None,
-                    repoid,
-                    commitid,
-                    branch,
-                    upload,
-                    framework,
-                    testruns,
-                )
-
-                if bq_enabled:
+
+            pg = PGDriver(repoid, db_session, flaky_test_set)
+            if settings.BIGQUERY_WRITE_ENABLED:
+                bq = BQDriver(repoid)
```

Comment:
instead of duplicating this code, how about taking advantage of the new driver interface you have introduced:

```
drivers = [pg, bq] if BQ_ENABLED else [pg] # or any kind of combination based on feature flags
for driver in drivers:
  driver.bulk_write_testruns(parsing_info)
```